wandb/sdk/service/service.py

```diff
@@ -31,25 +29,18 @@
 class ServiceStartProcessError(Error):
     """Raised when a known error occurs when launching wandb service."""
 
-    pass
-
 
 class ServiceStartTimeoutError(Error):
     """Raised when service start times out."""
 
-    pass
-
 
 class ServiceStartPortError(Error):
     """Raised when service start fails to find a port."""
 
-    pass
-
 
 class _Service:
     _settings: "Settings"
     _sock_port: Optional[int]
-    _service_interface: ServiceInterface
```

Comment:
we want to have the option to support grpc interface as well as a potential solution for multi-language

Reply:
that would be a detail in `service_connection.py`